assets/queries/cicd/github/unpinned_actions_full_length_commit_sha/query.rego

```diff
@@ -28,3 +29,8 @@ isPinned(use){
 	regex.match("@[a-f0-9]{40}$", use)
 }
 
+isRelative(use){
+	allowed := ["./", "../"]
```

Comment:
Hi @cw-alexcroteau.
First, thank you for your contribution :).
I'm with one doubt, relative to the "../".
Why do you feel the necessity to support the "../" ?